docs/BR.md

```diff
@@ -1190,15 +1190,36 @@ The certificate request MAY include all factual information about the Applicant
+
+Table: Maximum Allowed Non-SAN Validation Data Reuse Period
+
+| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |
+| --                                 | -                             | -                                         |
+|                                    | September 15, 2025            | 825                                       |
+| September 15, 2025                 | September 15, 2026            | 366                                       |
+
+For validation of Domain Names and IP Addresses according to [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address), any data, document, or completed validation used MUST be obtained within the maximum number of days days prior to issuing the Certificate, as defined in the following table:
+
+Table: Maximum Allowed SAN Validation Data Reuse Period
+
+| __Certificate issued on or after__ | __Certificate issued before__ | __Maximum number of days for data reuse__ |
+| --                                 | -                             | -                                         |
+|                                    | September 15, 2025            | 398                                       |
+| September 15, 2025                 | September 15, 2026            | 200                                       |
+| September 15, 2026                 | April 15, 2027                | 100                                       |
+| April 15, 2027                     | September 15, 2027            | 45                                        |
+| September 15, 2027                 |                               | 10                                        |
 
 In no case may a prior validation be reused if any data or document used in the prior validation was obtained more than the maximum time permitted for reuse of the data or document prior to issuing the Certificate.
 
-After the change to any validation method specified in the Baseline Requirements or EV Guidelines, a CA may continue to reuse validation data or documents collected prior to the change, or the validation itself, for the period stated in this BR 4.2.1 unless otherwise specifically provided in a ballot.
+After the change to any validation method specified in the Baseline Requirements or EV Guidelines, a CA may continue to reuse validation data or documents collected prior to the change, or the validation itself, for the period stated in this [Section 4.2.1](#421-performing-identification-and-authentication-functions) unless otherwise specifically provided in a ballot.
```

Comment:
```suggestion
After the change to any validation method specified in the Baseline Requirements or EV Guidelines, a CA may continue to reuse validation data or documents collected prior to the change, or the validation itself, for the period stated in [Section 4.2.1](#421-performing-identification-and-authentication-functions) unless otherwise specifically provided in a ballot.
```